packages/format-library/src/text-color/style.scss

```diff
@@ -1,7 +1,7 @@
 .components-inline-color-popover {
 
 	.components-popover__content {
-		.components-tab-panel__tab-content {
+		[role="tabpanel"] {
```

Comment:
We can probably clean up some of the styles in this file since they don't seem to apply anymore to the contents of the inline color popover. We should also change the `components-inline-color-popover` classname, since it could confuse folks that this is a component from the `@wordpress/components` package.

<details>

<summary>Here's a proposed diff</summary>

```diff
diff --git a/packages/format-library/src/text-color/inline.js b/packages/format-library/src/text-color/inline.js
index 3c34b760a2..98ced2cca0 100644
--- a/packages/format-library/src/text-color/inline.js
+++ b/packages/format-library/src/text-color/inline.js
@@ -166,7 +166,7 @@ export default function InlineColorUI( {
 	return (
 		<Popover
 			onClose={ onClose }
-			className="components-inline-color-popover"
+			className="format-library__inline-color-popover"
 			anchor={ popoverAnchor }
 		>
 			<Tabs>
diff --git a/packages/format-library/src/text-color/style.scss b/packages/format-library/src/text-color/style.scss
index 94a40b2c9b..439af6db38 100644
--- a/packages/format-library/src/text-color/style.scss
+++ b/packages/format-library/src/text-color/style.scss
@@ -1,23 +1,6 @@
-.components-inline-color-popover {
+.format-library__inline-color-popover {
 
-	.components-popover__content {
-		[role="tabpanel"] {
-			padding: 16px;
-		}
-
-		.components-color-palette {
-			margin-top: 0.6rem;
-		}
-
-		.components-base-control__title {
-			display: block;
-			margin-bottom: 16px;
-			font-weight: 600;
-			color: #191e23;
-		}
-
-		.component-color-indicator {
-			vertical-align: text-bottom;
-		}
+	[role="tabpanel"] {
+		padding: 16px;
 	}
 }


```

</details>